fastlane/Fastfile

```diff
@@ -396,6 +411,27 @@ lane :generate_fluent_file_from_po do |file_path:|
   fluent_file_path
 end
 
+desc 'Download the development signing certificates to this machine'
+lane :set_up_signing do |readonly: true|
+  require_env_vars!(*ASC_API_KEY_ENV_VARS, *CODE_SIGNING_STORAGE_ENV_VARS)
+
+  sync_code_signing(
+    platform: 'macos',
+    app_identifier: APPLE_BUNDLE_IDENTIFIER,
+    team_id: APPLE_TEAM_ID,
+    api_key: app_store_connect_api_key,
+    type: 'development',
+    certificate_id: 'Apple Development: Created via API (886NX39KP6)',
```

Comment:
What do you think about using the "Apple Distribution" one instead? I feel like it's appropriate to use binaries that are signed with "Apple Distribution" on all builds, but "Apple Development" builds may not be suitable for release builds.